bundles/org.openhab.binding.shelly/README.md

```diff
@@ -49,7 +49,7 @@ The binding provides the same feature set across all devices as good as possible
 | shellydimmer      | Shelly Dimmer                                          | SHDM-1    |
 | shellydimmer2     | Shelly Dimmer2                                         | SHDM-2    |
 | shellyix3         | Shelly ix3                                             | SHIX3-1   |
-| shellyuni         | Shelly UNI                                             | SHUNI-1   |
+| shellyuni         | Shelly UNI, Shelly Plus UNI                            | SHUNI-1, SNSN-0043X |
```

Comment:
Consider reformatting the table, since it was already formatted with vertical alignment.

Reply:
done